app/views/ops/_settings_server_tab.html.haml

```diff
@@ -269,7 +269,7 @@
         = _("Mode")
       .col-md-8
         = select_tag('webservices_mode',
-                      options_for_select(%w(invoke disable), @edit[:new][:webservices][:mode]),
+                      options_for_select([_("invoke"), _("disable")], @edit[:new][:webservices][:mode]),
```

Comment:
This change would mean that the form would submit localized strings as values, right?